SEBrowser/Web.config

```diff
@@ -6,7 +6,7 @@
   <categorizedSettings>
     <systemSettings>
       <!-- *** MODIFY CONNECTION STRING AND DATA PROVIDER STRINGS BELOW *** -->
-      <add name="ConnectionString" value="Data Source=localhost; Initial Catalog=OpenXDA; Integrated Security=SSPI" description="Configuration connection string" encrypted="false" />
+      <add name="ConnectionString" value="Data Source=vmtenantdb; Initial Catalog=TVAOpenXDA; Integrated Security=SSPI" description="Configuration connection string" encrypted="false" />
```

Comment:
Changes to config files don't need to be pushed

Reply:
As an aside: Changes to Config files SHOULD NOT BE PUSHED. it is a potential risk to expose database credentials. In this case not a big deal since the database can only be reached internally and no user is supplied but better practice is to not commit config files.

Reply:
@clackner-gpa Maybe we should add config files to the git ignore? It would still pull the file from git but it wouldn't add it to the commit if you did something like git add .

Reply:
Changes to files that are already added to the repo will always be staged by `git add .` regardless of whether they match a `.gitignore` rule. The only way I know of to avoid this is to be diligent and double-check your changes before committing or pushing.

Reply:
You are correct, but there is a local setting you can do. If you do the following command
git update-index --assume-unchanged [file]
you can tell get to assume that this file is unchanged. I don't think you can push this configuration to remote but it might help with local things.